### PR TITLE
Use case handler

### PR DESCRIPTION
**Pull request by alexfigner**
Added implementation in UCH for all currently added UseCases.

### CODE REVIEW

File: src/main/java/business/rules/UseCaseHandler.java [hunk 1]
```diff
@@ -41,6 +41,57 @@ public void handle(USE_CASE uc_id, Object[] data){
...
+        if(uc_id == USE_CASE.USER_LOGIN_USECASE){
+            this.uc = new UserLoginUseCase();
+            this.ucrq = new UseCaseLoginRequest(1, (String)data[0], (String)data[1], this.presenter.getUserDB());
+            UseCaseStringResponse resp = (UseCaseStringResponse) uc.process(ucrq);
+            if (resp.rCode == UseCaseResponse.RETURN_CODE.FAILURE) {
+                System.out.print(resp.getStr());
+                return;
+            }
+        }
+        if(uc_id == USE_CASE.USER_LOGIN_USECASE){
```
Comment (jaylovegood) [inline src/main/java/business/rules/UseCaseHandler.java]: should be USER_LOGOUT_USECASE

File: src/main/java/business/rules/UseCaseHandler.java [hunk 2]
```diff
@@ -41,6 +41,57 @@ public void handle(USE_CASE uc_id, Object[] data){
...
+            int i;
+            for (i = 0; i<3; i++) {
+                UseCaseStringResponse resp = (UseCaseStringResponse) uc.process(ucrq);
+                if (resp.rCode == UseCaseResponse.RETURN_CODE.FAILURE) {
+                    System.out.print(resp.getStr());
+                    return;
+                }
+            }
+        }
+        if(uc_id == USE_CASE.REMIX_RECIPE_USECASE){
```
Comment (jaylovegood) [inline src/main/java/business/rules/UseCaseHandler.java]: change `if`s to `else if`s?

**Review by jaylovegood — CHANGES_REQUESTED** [review]
Looks good, great work!
For the (minor) changes I'm requesting, please check my 2 comments :)
Also, AddToGroceriesUseCase has been implemented after this, so that would have to be added to the handler as well at some point.

**Review by jaylovegood — DISMISSED** [review]
Great work, thanks! Looks ready to merge after resolving conflicts

**Review by kumkum1 — APPROVED** [review]
Yup looks good!! Ready to merge

**Pull request merged**